media/stylus/base/utilities.styl

```diff
@@ -148,3 +148,15 @@ media-stack() {
     }
 }
 
+/* elements with this class move spans off screen and only display icons */
+.only-icon {
+    {$selector-icon} {
+        margin-left: 0 !important;
+        margin-right: 0 !important;
+        margin-top: -1px !important;
```

Comment:
Would it be possible to achieve the same effect with selector specificity? That would allow us to override this later if needed.

Maybe `html .only-icon {$selector-icon}` or similar?